lib/css/CssParser.js

```diff
@@ -1687,12 +1688,30 @@ class CssParser extends Parser {
 
 		const buildMeta = /** @type {BuildMeta} */ (state.module.buildMeta);
 
-		buildMeta.exportType = this.exportType;
 		buildMeta.exportsType = this.namedExports ? "namespace" : "default";
 		buildMeta.defaultObject = this.namedExports ? false : "redirect-warn";
+		buildMeta.exportType = this.exportType;
+
+		if (!buildMeta.exportType) {
+			// Inherit exportType from parent module to ensure consistency.
+			// When a CSS file is imported with syntax like `import "./basic.css" with { type: "css" }`,
+			// the parent module's exportType is set to "css-style-sheet".
+			// Child modules imported via @import should inherit this exportType
+			// instead of using the default "link", ensuring that the entire
+			// import chain uses the same export format.
+			const parent = state.compilation.moduleGraph.getIssuer(module);
```

Comment:
What about to use parent module, because issuer not always can be (although the parent too, but it seems to me more intuitive than taking the issuer)